readthedocs/proxito/views/serve.py

```diff
@@ -389,15 +378,15 @@ def get(self, request, proxito_path, template_name='404.html'):
                     self._register_broken_link(
                         project=final_project,
                         version=version,
-                        path=redirect_filename,
+                        path=filename,
                         full_path=proxito_path,
                     )
                     return resp
 
         self._register_broken_link(
             project=final_project,
             version=version,
-            path=redirect_filename,
+            path=filename,
```

Comment:
We already normalize the filename before recording the broken link, so we are fine using the original filename